index.js

```diff
@@ -31,5 +31,9 @@ var methods = {
 
 module.exports = function(filepath, cb) {
   var method = arguments.length === 2 ? 'async' : 'sync';
```

Comment:
Can you change this? It could be more faster evaluation asking for cb value 

var method = cb ? 'async' : 'sync'